.travis.yml

```diff
@@ -8,15 +6,10 @@ services:
   - docker
 
 before_install:
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
+  - pip install docker-compose
```

Comment:
Since each repo which uses the ci-deployment scripts needs docker-compose is it worth creating a `./scripts/ci-deployment/bootstrap.sh` script with the install in it. If we did that we could also document the reason why we explicitly install docker-compose rather than adding the Travis docker service (https://docs.travis-ci.com/user/docker/).

Also, we previously installed a specific version ($DOCKER_COMPOSE_VERSION) but now I think we just install the latest version. I presume that is intentional.